finn/finn_def.rb

```diff
@@ -0,0 +1,35 @@
+#practice using functions
+
+def prompt(message)
+	puts "#{message}"
+	answer = gets.chomp
+	puts "Thank you"  
+	return answer
+end
+
+def display_name(name)
+	puts "Good day. Your name is #{name},"
+end
+
+def display_year_born(year_born)
+	puts "You were born in #{year_born},"
+end
+
+def display_new_age(year_born)
+	new_age = 2017 - year_born.to_i - 5
```

Comment:
Instead of value 5, create a Constant variable and set it to the value of 5. Then whenever you going to reference the ''5 years ago' instead of using the value 5, use the Constant variable instead. e.g, year_born.to_i - FIVE_YEARS_AGO